### DISCUSSION TITLE
参考文献引用格式

### DISCUSSION THREAD

**Discussion by aperoe** [stone-zeng/fduthesis#288] [Q&A] [answered]
请问参考文献引用支持下面的格式么？谢谢！
![e6bd901b3bc0f772b710a38bcd23927](https://user-images.githubusercontent.com/120776777/211153998-e6256407-7b4f-41aa-8d4e-2afbf6a629a0.png)

**Comment by stone-zeng** [comment, marked as answer]
以下方法需要设置 `bib-backend = bibtex` 并使用 BibTeX 处理参考文献。导言区定义 `\numcite` 命令以输出不带括号的引用：

```tex
\makeatletter
\AtBeginDocument{
  \DeclareRobustCommand{\numcite}{%
    \begingroup
    \bibstyle@numbers
    \let\NAT@ctype\z@
    \NAT@parfalse
    \NAT@swatrue
    \@ifstar{\NAT@fulltrue\NAT@cites}{\NAT@fullfalse\NAT@cites}}
}
\makeatother
```

> 说明：这其实是在 `natbib` 包提供的 `\cite` 命令之上做了一些改动。其中 `\bibstyle@numbers` 表示不使用上标，`\NAT@parfalse` 表示不输出括号。

引用时按如下格式：

```tex
根据 [\numcite{...}，定理 3.3.5]，
```

如需使用斜体（意大利体），可以在外面加上 `\emph` 或 `\textit`。

**Reply by aperoe** [reply]
收到，感谢！不过就是为什么我编译出来是这样的呢？括号后面会换一行.....谢谢！！
![1673253880008](https://user-images.githubusercontent.com/120776777/211269228-457a3edf-6fb4-423b-8c2e-9f544aa71e24.png)

**Reply by stone-zeng** [reply]
给一段能编译的代码看下？

**Reply by aperoe** [reply]
不好意思，已解决，是我在复制代码的时候不小心加了空行..... 谢谢！！！
